src/main/java/cn/nukkit/block/BlockPumpkin.java

```diff
@@ -1,20 +1,30 @@
 package cn.nukkit.block;
 
 import cn.nukkit.Player;
+import cn.nukkit.block.BlockCarvedPumpkin;
+import cn.nukkit.blockproperty.BlockProperties;
 import cn.nukkit.item.Item;
 import cn.nukkit.item.ItemBlock;
+import cn.nukkit.item.ItemID;
+import cn.nukkit.item.ItemSeedsPumpkin;
```

Comment:
This import (cn.nukkit.item.ItemSeedsPumpkin) is not being used. Could you remove it?